build/templates/_library_interpreter.py.mako

```diff
@@ -0,0 +1,85 @@
+${template_parameters['encoding_tag']}
+# This file was generated
+<%
+import build.helper as helper
+
+config = template_parameters['metadata'].config
+attributes = config['attributes']
+
+module_name = config['module_name']
+c_function_prefix = config['c_function_prefix']
+driver_name = config['driver_name']
+
+functions = config['functions']
+functions = helper.filter_codegen_functions(functions)
+%>\
+
+import array
+import ctypes
+import hightime  # noqa: F401
+import ${module_name}._converters as _converters  # noqa: F401
+import ${module_name}._library_singleton as _library_singleton
+import ${module_name}._visatype as _visatype
+% if config['enums']:
+import ${module_name}.enums as enums
+% endif
+import ${module_name}.errors as errors
+% for c in config['custom_types']:
+
+import ${module_name}.${c['file_name']} as ${c['file_name']}  # noqa: F401
+% endfor
+
+
+# Helper functions for creating ctypes needed for calling into the driver DLL
+def get_ctypes_pointer_for_buffer(value=None, library_type=None, size=None):
+    if isinstance(value, array.array):
+        assert library_type is not None, 'library_type is required for array.array'
+        addr, _ = value.buffer_info()
+        return ctypes.cast(addr, ctypes.POINTER(library_type))
+    elif str(type(value)).find("'numpy.ndarray'") != -1:
+        import numpy
+        return numpy.ctypeslib.as_ctypes(value)
+    elif isinstance(value, bytes):
+        return ctypes.cast(value, ctypes.POINTER(library_type))
```

Comment:
I'm sure we already discussed this, but this case is new. Is it a bug fix?

Reply:
It's not new; you must've overlooked it. I just verified that this function is identical to what was removed from session.py.mako.

Reply:
I see what I did. I was comparing it with the nitclk/session.py.mako.

Reply:
I guess that means it is new for nitclk.

Reply:
Agreed, it's new for tclk. I don't see anything in tclk that would be of type `bytes`, though, so I don't think it'll have any effect there. (Note that even for other drivers, there's only bytes in the places where we use `convert_to_bytes`.)